simulator/util/consts.go

```diff
@@ -0,0 +1,3 @@
+package util
+
+const EtcdPrefix = "/kube-scheduler-simulator"
```

Comment:
It's weird to put in util... Can we create another package named `const` to store all this kind of static values?

Reply:
I agree with that it's weird. But if all kind of static values moves to `consts` package, I guess it becomes difficult for maintainers to read. Because, I think the each static value has already been located properly. But I don't have better idea to store this value to suitable place. Or this value is only used in reset package, so should we write in it ?